src/completion.rs

```diff
@@ -1654,17 +1644,19 @@ fn create_completion_package_removed(
         })
         .collect();
 
+    // XXX: rockstar (5 Feb 2022) - This is the cause of issue #391. This should
+    // bubble up and emit some diagnostic messages.
     match analyze(&pkg) {
         Ok(p) => Ok(p),
-        Err(e) => Err(format!("ERROR IS HERE {}", e)),
+        Err(_) => Err(()),
```

Comment:
If you are going to have `()` as an error type, you may as well just use `Option` instead?

Reply:
Yeah, that's what I was thinking, but there's a bunch of follow on effects that seemed to make the patch bigger. At least with `()` I have a marker that says "Hey, you don't need this anymore" when I get to that bigger refactor work.

Reply:
Oh, I beg your pardon. I misread what you were saying specific to where you were saying it. Yeah, that's a good change, which I've made.